src/components/ProductGalleryPopup/ProductGalleryPopup.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import styles from "./ProductGalleryPopup.module.css";
+import LeftArrowIcon from "../../icons/LeftArrowIcon";
+import ShareIcon from "../../icons/ShareIcon";
+import HeartIcon from "../../icons/HeartIcon";
+
+const ProductGalleryPopup = ({
+  showPopupHandler,
+  livingArea,
+  kitchen,
+  bathroom,
+  exterior,
+  Bedroom,
+}) => {
+  const galleryImgs = [livingArea, kitchen, bathroom, exterior, Bedroom];
+
+  function handleShare() {
+    alert("Share this experience");
+  }
+  function handleSave() {
+    alert("Save this experience");
+  }
+
+  return (
+    <div className={styles.popupMainCntainer}>
+      <div className={styles.header}>
+        <LeftArrowIcon showPopupHandler={showPopupHandler} />
+        <div className={styles.saveShareIconContainer}>
+          <div
+            className={styles.shareContainer}
+            onClick={() => {
+              alert("At the moment, this button is not linked!");
+            }}
+          >
+            <ShareIcon height={30} width={30} fill="black" />
+            <span className={styles.shareSaveText}>Share</span>
+          </div>
+          <div
+            className={styles.SaveContainer}
+            onClick={() => {
+              alert("At the moment, this button is not linked!");
+            }}
+          >
+            <HeartIcon height={20} width={20} fill="black" />
+            <span className={styles.shareSaveText}>Save</span>
+          </div>
+        </div>
+      </div>
+
```

Comment:
We already have a component for these exact buttons: IconButton

Reply:
In so many components, we have used SVG icons rather than font-awesome. Some of them are also available in the icons folder. I believe the performance, scalability, quality, customization, and accessibility of SVG icons are much higher than font-awesome. As we have svg icons for this component, I think we can later remove the font-awesome icons and replace these in other components.
You can take a look at this page for more information:
https://www.keycdn.com/blog/icon-fonts-vs-svgs#:~:text=Icon%20fonts%20can%20be%20cached,be%20cached%20by%20the%20browser.

Reply:
Ok, makes sense, maybe we could update the buttons then. I'd recommend looking again at the heart icon sizes --> at the moment the heart is cut at the bottom. 
![Screenshot 2024-10-08 at 15 15 43](https://github.com/user-attachments/assets/2a339f3b-f4a1-45df-b10d-38f1c2abbec3)

Reply:
Additionally, I noticed once I open the gallery pop-up, 2 scroll bars are visible: one for the gallery pop-up and another one for the product page.